datafusion/sqllogictest/test_files/string/init_data.slt.part

```diff
@@ -15,10 +15,12 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# --------------------------------------
-# Test `substr()` with literal arguments
-# --------------------------------------
-include ./substr_literal.slt.part
+statement ok
+create table test_source as values
+  ('Andrew', 'X', 'datafusion📊🔥', '🔥'),
+  ('Xiangpeng', 'Xiangpeng', 'datafusion数据融合', 'datafusion数据融合'),
+  ('Raphael', 'R', 'datafusionДатаФусион', 'аФус'),
+  (NULL, 'R', NULL, '🔥');
```

Comment:
Because we started to implement some ASCII-faster paths, I think we should always consider testing both ASCII-only and Unicode cases.